libphal/phal_exception.H

```diff
@@ -48,6 +49,8 @@ enum ERR_TYPE {
 	PDBG_FSI_WRITE_FAIL,
 	SBE_DUMP_IS_ALREADY_COLLECTED,
 	SBE_INTERNAL_FFDC_DATA,
+	INVALID_LOC_CODE,
```

Comment:
Why we need two enums. The enum `INVALID_LOC_CODE` can be applied to both the cases for `Given location code is not valid` and `Given location code is not found in device tree`, right? If a location code is not there in the device tree it is invalid as well. Isn't it?